Add acknowledgements section (#189)
decentralized-identity/bbs-signature: draft-bbs-signatures.md

This document does not make any requests of IANA.

# Acknowledgements

> The authors would like to acknowledge the significant amount of academic work that preceeded the development of this document. In particular the original work of [@SGS] which was subsequently developed in [@CDL16].

Reviewer: ```suggestion
The authors would like to acknowledge the significant amount of academic work that preceeded the development of this document. In particular the original work of [@BBS04] which was subsequently developed in [@ASM06] and in [@CDL16]. This last academic work is the one mostly used by this document.
```
As far as i know of, It's more customary to cite using the authors initials with the year of publishing (at least in this format).

Also added one more paper.

 <seriesInfo name="Springer," value="Cham"/>
</reference>

> <reference anchor="SGS" target="https://link.springer.com/chapter/10.1007/978-3-540-28628-8_3">

Reviewer: ```suggestion
<reference anchor="BBS04" target="https://link.springer.com/chapter/10.1007/978-3-540-28628-8_3">
```
Because of my comment above (citing using the authors initials).

 <front>
   <title>Short Group Signatures</title>
   <author initials="D." surname="Boneh" fullname="Dan Boneh">
    </author>
    <author initials="X." surname="Boyen" fullname="Xavier Boyen">
    </author>
    <author initials="H." surname="Shacham" fullname="Hovav Scacham">
    </author>
    <date year="2004"/>
 </front>
 <seriesInfo name="In" value="Advances in Cryptology"/>
 <seriesInfo name="pages" value="41-55"/>
</reference>

> 

Reviewer: ```suggestion

<reference anchor="ASM06" target="https://link.springer.com/chapter/10.1007/11832072_8">
 <front>
   <title>Constant-Size Dynamic k-TAA</title>
   <author initials="M. H." surname="Au" fullname="Man Ho Au">
    </author>
    <author initials="W." surname="Susilo" fullname="Willy Susilo">
    </author>
    <author initials="Y." surname="Mu" fullname="Yi Mu">
    </author>
    <date year="2006"/>
 </front>
 <seriesInfo name="In" value="International Conference on Security and Cryptography for Networks"/>
 <seriesInfo name="pages" value="111-125"/>
 <seriesInfo name="Springer," value="Berlin, Heidelberg"/>
</reference>
```
I think this should maybe be mentioned too. There is also the [[CL04]](https://link.springer.com/content/pdf/10.1007/978-3-540-28628-8_4.pdf) paper that came before, but it was the above that proved security and "popularized" BBS+ (this is the one that is referenced by [[CDL16]](https://eprint.iacr.org/2016/663.pdf) for example).
